mlir/utils/jenkins/Jenkinsfile

```diff
@@ -599,11 +599,15 @@ pipeline {
                         steps {
                             buildProject('check-rocmlir-build-only ci-performance-scripts', '')
                             dir('build') {
-                                // Tune gemms, fail if the DB is not created
+                                // Tune gemms with default datatypes, fail if the DB is not created
                                 sh """python3 ./bin/tuningRunner.py --operation gemm \
                                         --configs_file=../mlir/utils/performance/gemm-configs \
                                         --output=mlir_tuning_${ARCH}.tsv
                                       [ -f mlir_tuning_${ARCH}.tsv ]"""
+                                // Tune gemms int8xint8->int8
```

Comment:
Let's go ahead and update this with "for performance comparisons against CK. Ideally, this'll be removed in the future."